com.civicrm.hrjobroles/CRM/Hrjobroles/Upgrader.php

```diff
@@ -36,9 +36,11 @@ public function upgrade_1002(){
     return TRUE;
   }
```

Comment:
also can you modefiy upgrade_1004 or add another upgrader ( 1005 ) to update current job roles table (end_date) field and in case it is = 0000-00-00 or 0 then set it to null .

`UPDATE civicrm_jobroles SET end_date = NULL where (end_date = '0000-00-00 00:00:00' OR end_date = '0' ) `